egs/callhome_diarization/v2/run.sh

```diff
@@ -53,7 +70,7 @@ if [ $stage -le 1 ]; then
   # callhome1 and callhome2.  Each partition is treated like a held-out
   # dataset, and used to estimate various quantities needed to perform
   # diarization on the other part (and vice versa).
-  for name in train callhome1 callhome2; do
+  for name in train callhome callhome1 callhome2; do
```

Comment:
Is the dataset "callhome" just the combination of "callhome1" and "callhome2?" If so, it's wasteful to create features for the entire dataset, in addition to the two partitions. It would be better to create features for the two partitions and then use combine_data_dir.sh to combine them.

Reply:
> Added a few comments. Will add more later. For now, the main things are:
> 
> * General purpose scripts should go in diarization/, not local/. We might use these scripts for other recipes, e.g., DIHARD.
      ok I will move VB related scripts to a new folder - diarization.
> * I don't understand why the VB needs to be trained on different datasets than the ones we used to train the x-vector system. Seems to be complicating things too much.
      Yes, we used mx6 and sre10 data also in training ivector-extractor. So, shall we add these two sources in training x-vector system also?
> * Currently you're only reporting results using the oracle number of speakers. We definitely need to see results using a tuned stopping threshold, which is more realistic.
       Yes, with tuned stopping threshold, I get 8.21% DER before VB and 6.32% DER after VB.